aggregator_core/src/query_type.rs

```diff
@@ -427,17 +438,11 @@ impl CollectableQueryType for FixedSize {
     type Iter = iter::Once<Self::BatchIdentifier>;
 
     async fn collection_identifier_for_query<C: Clock>(
-        tx: &Transaction<'_, C>,
-        task: &Task,
+        _: &Transaction<'_, C>,
+        _: &Task,
         query: &Query<Self>,
     ) -> Result<Option<Self::BatchIdentifier>, datastore::Error> {
-        match query.fixed_size_query() {
-            FixedSizeQuery::ByBatchId { batch_id } => Ok(Some(*batch_id)),
-            FixedSizeQuery::CurrentBatch => {
-                tx.get_filled_outstanding_batch(task.id(), task.min_batch_size())
-                    .await
-            }
-        }
+        Ok(Some(*query.batch_id()))
```

Comment:
I suspect our partner will end up using fixed-size queries (just because they were involved in inventing them).

However, while DAP-02 does not have the concept of `CurrentBatch`, our partner can implement something proprietary to achieve the same effect, as they control both the Leader & the Collector. I suspect this is why no equivalent of `CurrentBatch` made it into DAP-02.